app/src/main/java/com/kylecombes/micrepeater/ui/main/AmplifyingControlTile.java

```diff
@@ -28,10 +31,12 @@ public AmplifyingControlTile(Context context, AttributeSet attrs) {
     public void setAmplifyingActive(boolean isActive) {
         if (isActive) {
             mStartStopButton.setText(R.string.stop);
-            mStatusTextView.setText(R.string.on);
+            mStatusTextView.setText(R.string.amplification_active);
+            mStatusTextView.setTextColor(COLOR_RED);
         } else {
             mStatusTextView.setText(R.string.off);
             mStartStopButton.setText(R.string.start);
```

Comment:
Somehow the text "START" for me got cut off, and it shows "STAR". Also, my phone cannot display battery level but somehow this happened.
![Screenshot_2020-06-21-18-16-18](https://user-images.githubusercontent.com/13646394/85240085-d7b42e00-b404-11ea-983f-6472ade9a5ed.png)

Reply:
That's because I specified a width on the button to make it be the same size regardless of what the text was. I'll undo that...